packages/marshal/marshal.js

```diff
@@ -154,11 +154,13 @@ function isPassByCopyError(val) {
   const { name } = val;
   const EC = getErrorConstructor(name);
   if (!EC || EC.prototype !== proto) {
-    throw TypeError(`Must inherit from an error class .prototype ${val}`);
+    throw TypeError(
+      `Errors must inherit from an error class .prototype ${val}`,
+    );
   }
 
   const {
-    message: { value: messageStr } = { value: '' },
+    message: mDesc,
     // Allow but ignore only extraneous own `stack` property.
     // TODO: I began the variable below with "_". Why do I still need
```

Comment:
You should be able to remove this TODO and the `eslint-disable-next-line no-unused-vars` comments now that you updated the lint rules.